vendor/BUILD.bazel

```diff
@@ -229,6 +230,71 @@ cc_library(
     ],
 )
 
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
```

Comment:
This includes all source code from all libraries in `vendor`.

Reply:
There is a problem here: Harfbuzz's header files can't pass the tidy-clang check and will raise warnings, cause we treat warnings as errors in the core, so I have to create a wrap header file to hide the warning on the vendor side. So I Can put freetype and harfbuzz under vendor and must create a warp directory to store my wrap files, is that acceptable?

Reply:
resettle the vendors.